src/PostBuildEvents.bat

```diff
@@ -0,0 +1,17 @@
+REM	PostBuildEvents.bat is called with the following parameters
+REM		%1 is TargetPath
+REM		%2 is TargetDir
+REM     %3 is ProjectName
+REM     %4 is ProjectDir
+
+REM	/S (Copies directories and subdirectories except empty ones)
+REM /Y (Suppresses prompting to confirm you want to overwrite an existing destination file.)
+
+REM Copy bin folder to website folder
+xcopy /s /y "%1" "{YOUR-WEBSITE-LOCATION}\bin\"
+
+REM Copy pdb
+xcopy /s /y "%2%3.pdb" "{YOUR-WEBSITE-LOCATION}\bin\"
+
+REM Copy package front-end files
+xcopy /s /y "%4Web\UI\*.*" "{YOUR-WEBSITE-LOCATION}"
```

Comment:
I'm loving this file, thank you! 🙏 

🤔 Wondering if we could use a local variable here for the `{YOUR-WEBSITE-LOCATION}`?
e.g. `SET /P WEBSITE_PATH=C:\path\to\my\umbraco8\website`

_I'll have a play around._ 😃

Reply:
@leekelleher would love to see if this could be a bit easier!

Reply:
Just added this in.